docs/siem/detections/api/exceptions/api-create-exception-item.asciidoc

```diff
@@ -0,0 +1,318 @@
+
+Creates an exception item and associates it with the specified
+<<exceptions-api-create-container, exception container>>.
+
+See <<lists-api-overview>> for information about creating exception items from
+lists, such as a list of IP addresses or host names.
+
+NOTE: Before creating exception items, you must create an exception container.
+
+[IMPORTANT]
+==============
+Endpoint rule exception items cannot use
+<<lists-api-overview, lists>> (the `list` element in the `entries` object), and
+the following fields cannot be used in exception queries (as `field` values in
+the `entries` object):
+
+* `file.Ext.quarantine_path`
+* `file.Ext.quarantine_result`
+* `process.entity_id`
+* `process.parent.entity_id`
+* `process.ancestry`
+==============
+
+==== Request URL
+
+`POST <kibana host>:<port>/api/exception_lists/items`
+
+==== Request body
+
+A JSON object with these fields:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`comments` |comments[] a|Array of `comment` fields:
+
+* `comment` (string): Comments about the exception item.
+
+|No, defaults to empty array.
+
+|`description` |String |Describes the exception item. |Yes
+|`entries` |<<entries-object-schema, entries[]>> |Array containing the
+exception queries. Boolean `AND` logic is used to evaluate the relationship
+between array elements. If you want to use `OR` logic, create a separate
+exception item. |No, defaults to an empty `entries` array.
+|`list_id` |String |ID of the associated <<exceptions-api-create-container, exception container>>. |Yes
+|`item_id` |String |Unique identifier of the exception item. |No, automatically
+created when it is not provided.
+|`meta` |Object |Placeholder for metadata about the exception item. |No
+|`name` |String |The exception item's name. |Yes.
+|`namespace_type` |String a|Determines whether the exception item is available
+in all {kib} spaces or just the space in which it is created, where:
+
+* `single`: Only available in the {kib} space in which it is created.
+* `agnostic`: Available in all {kib} spaces.
+
+|No, defaults to `single`.
+|`tags` |String[] |String array containing words and phrases to help categorize
+exception items. |No 
+|`type` |String a|Exception query type, must be `simple`. |Yes
+|`_tags` |String[] a|For endpoint rules only, defines the OS on which the
+exception is implemented. Valid values are:
+
+* `os:windows`: Windows OS
+* `os:linux`: Linux OS
+* `os:macos`: Mac OS
+
+The array must also include an `endpoint` element (to implement the exception on Linux hosts, use: `["endpoint", "os:linux"]`).
+
+|For endpoint exceptions, yes. For detection exceptions, no.
+
+|==============================================
+
+[[entries-object-schema]]
+===== `entries` schema
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`field` |String |The field used to define the exception. |Yes
```

Comment:
Do you think it might be helpful to separate these out more clearly into the allowed fields for each type (`match`, `match_any`, etc). 

Also just a note that the `field` is now typed to a non empty string so has to have a value. This is also true of `value`. For `match`, `value` must be a non empty string. For `match_any`, `value` must be a non empty string array. For `nested`, the `entries` field must be a non empty array of `entries` (that exclude lists or nested types - ie: not recursive structure).

Reply:
@yctercero - I'm not sure what limitations apply to the `field` value per `operator`. Can you explain? Thanks.